libs/settings/api/src/lib/settings.resolver.spec.ts

```
@@ -141,6 +131,31 @@ jest.mock('@prisma/client', () => {
+    SettingsService,
+    GraphQLSettingValueType,
+    {
+      provide: SettingDataloaderService,
+      useValue: {
+        load: () => mockSettingFindUnique,
+        prime: jest.fn()
+      }
+    }
+  ]
+})
+export class AppModule {}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)